certgen/testdata/test_ca.pem

```diff
@@ -0,0 +1,83 @@
+Certificate:
+    Data:
+        Version: 3 (0x2)
+        Serial Number:
+            67:94:a1:b8:a0:54:22:26:c8:c4:a4:46:c4:42:9a:95:32:b6:02:39
+        Signature Algorithm: sha256WithRSAEncryption
+        Issuer: C = GB, ST = Test State, L = Test Locality, O = Test Organisation, OU = Test Organizational Unit Name, CN = Test CA
+        Validity
+            Not Before: Sep 10 10:11:57 2019 GMT
+            Not After : Oct 10 10:11:57 2019 GMT
```

Comment:
Is this going to cause trouble running the test in the very near future, after 10 Oct 2019?
Hopefully not as you pin time.Now() to be 25 Mar 2019.

Reply:
Tbh, I have absolutely no idea!  I wonder if the x509.CreateCertificate code checks the validity of the cert...

This was what was generated when I created a new self-signed cert using openssl.  I didn't even look at the validity period it used.